internal/pkg/service/appsproxy/proxy/proxy_test.go

```diff
@@ -289,22 +292,23 @@ func TestAppProxyRouter(t *testing.T) {
 				require.NoError(t, err)
 				require.Equal(t, http.StatusFound, response.StatusCode)
 				cookies := response.Cookies()
-				assert.Len(t, cookies, 2)
-
-				assert.Equal(t, "_oauth2_provider", cookies[0].Name)
-				assert.Equal(t, "", cookies[0].Value)
-				assert.Equal(t, "/", cookies[0].Path)
-				assert.Equal(t, "oidc.hub.keboola.local", cookies[0].Domain)
-				assert.True(t, cookies[0].HttpOnly)
-				assert.True(t, cookies[0].Secure)
-
-				assert.Equal(t, "_oauth2_proxy", cookies[1].Name)
-				assert.Equal(t, "", cookies[1].Value)
-				assert.Equal(t, "/", cookies[1].Path)
-				assert.Equal(t, "oidc.hub.keboola.local", cookies[1].Domain)
-				assert.True(t, cookies[1].HttpOnly)
-				assert.True(t, cookies[1].Secure)
-				assert.Equal(t, http.SameSiteLaxMode, cookies[1].SameSite)
+				if assert.Len(t, cookies, 2) {
+					assert.Equal(t, "_oauth2_provider", cookies[0].Name)
+					assert.Equal(t, "", cookies[0].Value)
+					assert.Equal(t, "/", cookies[0].Path)
+					assert.Equal(t, "oidc.hub.keboola.local", cookies[0].Domain)
+					assert.True(t, cookies[0].HttpOnly)
+					assert.True(t, cookies[0].Secure)
+					assert.Equal(t, http.SameSiteLaxMode, cookies[0].SameSite)
+
+					assert.Equal(t, "_oauth2_proxy", cookies[1].Name)
+					assert.Equal(t, "", cookies[1].Value)
+					assert.Equal(t, "/", cookies[1].Path)
+					assert.Equal(t, "oidc.hub.keboola.local", cookies[1].Domain)
+					assert.True(t, cookies[1].HttpOnly)
+					assert.True(t, cookies[1].Secure)
+					assert.Equal(t, http.SameSiteLaxMode, cookies[1].SameSite)
+				}
```

Comment:
```suggestion
				assert.Len(t, cookies, 2)
				oauthProviders := []string {
				     "_oauth2_provider",
				     "_oauth2_proxy",
				}
				for i, cookie := range cookies {
					assert.Equal(t, oauthProviders[i], cookie.Name)
					assert.Equal(t, "", cookie.Value)
					assert.Equal(t, "/", cookie.Path)
					assert.Equal(t, "oidc.hub.keboola.local", cookie.Domain)
					assert.True(t, cookie.HttpOnly)
					assert.True(t, cookie.Secure)
					assert.Equal(t, http.SameSiteLaxMode, cookie.SameSite)
				}
```
Could be copied also below in another tests. When you add cookie you copy boilerplate. I think this is just improvement for future cookie extension and also prescription how to test multiple cookies in future.

Reply:
Yes, the test may be improved, I did not make big changes there so that changes in behavior could be seen.